plugins/pencilblue/controllers/section.js

```diff
@@ -39,32 +39,32 @@ module.exports = function(pb) {
         var init = function(err) {
             //get content settings
             var serviceContext = self.getServiceContext();
-            var contentService = new pb.ContentService({site: self.site, onlyThisSite: serviceContext.onlyThisSite});
+            var contentService = new pb.ContentService(context);
```

Comment:
I think you want to pass the service context.  The reason you call the ```getServiceContext``` is because it creates a separate object to pass to each service.  This prevents the individual services from modifying the context object and causing issues across other services.